pkgs/applications/window-managers/herbstluftwm/default.nix

```diff
@@ -1,23 +1,22 @@
-{ stdenv, fetchurl, pkgconfig, glib, libX11, libXext, libXinerama }:
+{ stdenv, fetchurl, cmake, pkgconfig, glib, libX11, libXext, libXinerama, libXrandr, asciidoc }:
 
 stdenv.mkDerivation rec {
-  name = "herbstluftwm-0.7.2";
+  pname = "herbstluftwm";
+  version = "0.8.0";
 
   src = fetchurl {
-    url = "https://herbstluftwm.org/tarballs/${name}.tar.gz";
-    sha256 = "1kc18aj9j3nfz6fj4qxg9s3gg4jvn6kzi3ii24hfm0vqdpy17xnz";
+    url = "https://herbstluftwm.org/tarballs/herbstluftwm-${version}.tar.gz";
+    sha256 = "04n8cmrgb490kfn70x9kgxp286mrrn7nb4irhi7nvbm6lv0z28sq";
   };
 
-  patchPhase = ''
-    substituteInPlace config.mk \
-      --replace "/usr/local" "$out" \
-      --replace "/etc" "$out/etc" \
-      --replace "/zsh/functions/Completion/X" "/zsh/site-functions" \
-      --replace "/usr/share" "\$(PREFIX)/share"
+  outputs = [ "out" "doc" "man" ];
+
+  preConfigure = ''
```

Comment:
I think it's more idiomatic to do the following:

```
cmakeFlags = [
    "-DCMAKE_INSTALL_SYSCONF_PREFIX=$out/etc"
];
```

(Make sure it actually does what it should, though; I could be wrong.)